locales/th/json.json

```diff
@@ -59,19 +59,19 @@
     "April": "เอพริล",
     "Are you sure you want to delete the selected resources?": "คุณแน่ใจหรือว่าคุณต้องการที่จะลบที่เลือกไว้แล้วเหรอครับ?",
     "Are you sure you want to delete this file?": "คุณแน่ใจหรือว่าคุณต้องการจะลบแฟ้มนี้?",
-    "Are you sure you want to delete this notification?": "Are you sure you want to delete this notification?",
+    "Are you sure you want to delete this notification?": "คุณแน่ใจหรือว่าคุณต้องการจะลบการแจ้งเตือนนี้?",
     "Are you sure you want to delete this resource?": "คุณแน่ใจหรือว่าคุณต้องการจะลบเรื่องทรัพยากรได้?",
     "Are you sure you want to delete this team? Once a team is deleted, all of its resources and data will be permanently deleted.": "คุณแน่ใจหรือว่าคุณต้องการจะลบทีมนี้? ครั้งนึงเป็นทีมเป็นลบทั้งหมดของมันทรัพยากรและข้อมูลจะถูกลบออกไปอย่างถาวร.",
     "Are you sure you want to delete your account? Once your account is deleted, all of its resources and data will be permanently deleted. Please enter your password to confirm you would like to permanently delete your account.": "คุณแน่ใจหรือว่าคุณต้องการจะลบบัญชีของคุณ? เมื่อบัญชีของคุณคือกลบออกไป,ทั้งหมดของทรัพยากรและข้อมูลจะถูกลบออกไปอย่างถาวร. โปรดป้อนรหัสผ่านของคุณเพื่อยืนยันว่าคุณจะชอบอย่างถาวรลบบัญชีของคุณ.",
     "Are you sure you want to detach the selected resources?": "คุณแน่ใจหรือว่าต้องการที่จะยกเลิกแนบรางที่เลือกไว้แล้วเหรอครับ?",
     "Are you sure you want to detach this resource?": "คุณแน่ใจหรือว่าต้องการที่จะยกเลิกแนบรางเรื่องทรัพยากรได้?",
     "Are you sure you want to force delete the selected resources?": "คุณแน่ใจหรือว่าต้องการจะบังคับให้ลบที่เลือกไว้แล้วเหรอครับ?",
     "Are you sure you want to force delete this resource?": "คุณแน่ใจหรือว่าต้องการจะบังคับให้ลบเรื่องทรัพยากรได้?",
-    "Are you sure you want to log out?": "Are you sure you want to log out?",
+    "Are you sure you want to log out?": "คุณแน่ใจหรือว่าต้องการลงชื่อออกจากระบบ",
     "Are you sure you want to restore the selected resources?": "คุณแน่ใจหรือว่าต้องการจะเรียกคืนข้อมูลที่เลือกไว้แล้วเหรอครับ?",
     "Are you sure you want to restore this resource?": "คุณแน่ใจหรือว่าต้องการจะลบทรัพยากร?",
     "Are you sure you want to run this action?": "คุณแน่ใจนะว่าคุณต้องการประมวลผลการกระทำนี้?",
-    "Are you sure you want to stop impersonating?": "Are you sure you want to stop impersonating?",
+    "Are you sure you want to stop impersonating?": "คุณแน่ใจหรือว่าต้องการหยุดการแอบอ้างนี้",
```

Comment:
From a grammatical point of view, is your sentence filled out correctly, given the lack of a question mark at the end?

Reply:
Thai does not need the use of a question mark, but we can use one to emphasize that this line is a question. I've updated this one.

Reply:
Understood, thanks.